scripts/get-versions.js

```diff
@@ -68,7 +73,7 @@ const main = async () => {
     .filter((tag) => !tag.name.match(/sdk|pkg/))
     .map((tag) => ({
       version: tag.name,
-      date: tag.target.committedDate,
+      date: tag.target?.committedDate || tag.target?.tagger?.date,
```

Comment:
So this essentially says, "use `target.committedDate` if possible, otherwise fall back to `tag.tagger.date`"? But if that's not there, this may still be `undefined`, right?

Reply:
Yes, but from what I understand that shouldn't be possible. Haven't found solid documentation on it yet.